modular_skyrat/master_files/code/modules/clothing/under/jobs/civilian/civilian.dm

```diff
@@ -47,3 +47,101 @@
 	body_parts_covered = CHEST|GROIN|ARMS
 	dying_key = DYE_REGISTRY_JUMPSKIRT
 	female_sprite_flags = FEMALE_UNIFORM_TOP_ONLY
+
+/*
+*     BARTENDER
+*/
+
+/obj/item/clothing/under/civilian/bartender/sleeky
```

Comment:
on **line 7**, please add:
```code
/obj/item/clothing/under/rank/civilian/bartender/skyrat
	icon = 'modular_skyrat/master_files/icons/obj/clothing/under/civilian.dmi'
	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/under/civilian.dmi'
	can_adjust = FALSE
```

Then, you can instead use
```suggestion
/obj/item/clothing/under/civilian/bartender/skyrat/sleeky
```
and with this, you won't have to re-define the variables for icon, worn_icon, and can_adjust